### ISSUE TITLE
duplicate

### ISSUE THREAD

**Issue by zack-bitcoin** [demiculus/public#1]
you list Aeternity twice.

**Comment by demiculus** [comment]
True, because each coin is researched by minimum of 3 people. I just put enough notes to show what we're doing. We're not ready to open source all of our research.

**Comment by Esqarrouth** [comment]
@zack-bitcoin 

How accurate do you think notes on Aeternity are?
